app/models/chargeback_container_project.rb

```diff
@@ -62,19 +64,38 @@ def self.build_results_for_report_ChargebackContainerProject(options)
       @data_index.store_path(:container_project, :by_group_id, g.id, g.container_project || g.old_container_project)
     end
 
+    if @options[:groupby_tag]
+      @tags_for_category = Classification.find_by_name(@options[:groupby_tag]).entries
+      @tags_for_category.each do |tag|
+        ContainerProject.with_any_tags([tag.tag_id]).each do |project|
+          @data_index.store_path(:tag, :by_project_id, project.id, tag)
+        end
+      end
+    end
```

Comment:
ugh. this seems like a simple question.
And I'm not totally sure why you are storing tag in the index. I guess I was expecting you to store container projects in there.

unworking/imaginary code:
```ruby
ContainerProject.with_any_tags(@tags_for_category).group_by_tags_in_category(@classification).group_by(:tag).each do |tag, project|
  # yield
end
```

- looks like `with_any_tags` is pretty bad currently. should be a normal where clause
- bringing back all the tags for the project is overkill and typically an N+1. we have a number of places where we want tags just in a category.

I'll try and think about this, but plate is full. if you have ideas, please share